docs/articles/expensify-classic/workspace-and-domain-settings/Categories.md

```diff
@@ -1,5 +1,137 @@
+## Add sub-categories
+Sub-categories are useful when you want to be more specific, i.e. Travel could have a subcategory of airplane or lodging so the outcome would be Travel:airplane or Travel:lodging.
+If you would like to create sub-categories under your category selection drop-down list, you can do so by adding a colon after the name of the desired category and then typing the sub-category (without spaces around the punctuation).
+# How to import Categories with an accounting integration connected
+If you connect Expensify to a direct integration such as QuickBooks Online, QuickBooks Desktop, Sage Intacct, Xero, or NetSuite, then Expensify automatically imports the general ledger parameters from your accounting system as Categories.
+
+When you first connect your accounting integration your categories will most likely be pulled from your chart of accounts, however this can vary depending on the account integration. 
+
+If you need to update your categories in Expensify, you will first need to update them in your accounting system, then sync the connection in Expensify by navigating to Settings > Workspace > Group > [Workspace Name] > Connection > Sync Now.
+
+Alternatively, if you update the category details in your accounting integration, be sure to sync the policy connection so that the updated information is available on the workspace. 
+# Deep Dive
+## Category-specific rules and description hints
+If you're an admin using a workspace on a Control plan, you have the ability to enable specific rules for each category.
+
+These settings are valuable if you want to set a special limit for a certain category. e.g. Your default expense limit is $2500 but Entertainment has a limit of $150 per person, per day. You can also require or not require receipts, which is great for allowing things like Mileage or Per Diems to have no receipt.
+
+To set up Category Rules, go to Settings > Workspace> Group > [Workspace Name] > Categories.
+
+Then, click Edit Rules next to the category name for which you'd like to define a rule.
+
+- GL Code and Payroll Code: These are optional fields if these categories need to be associated with either of these codes in your accounting or payroll systems. GL code will be automatically populated if connecting to an accounting integration
+- Max Amount: Allows you to set specific expense amount caps based on the expense category. Using Limit type, you can define this per individual expense, or per day (for expenses in a category on an expense report).
+- Receipts: Allows you to decide whether you want to require receipts based on the category of the expense. For instance, it's common for companies to disable the receipt requirement for Toll expenses
+- Description: Allows you to decide whether to require the description field to be filled out based on the category of the expense
```

Comment:
Missing formatting:
**Description** and `description` field

Reply:
**description** field is off. It's in bold whereas it should be like this: `description`.

Reply:
Resolved